R/arc-open.R

```diff
@@ -53,10 +56,40 @@
 #' )
 #'
 #' arc_open(map_url)
+#'
+#' # Item Page (Feature Service or Map Service)
+#'
+#' item_url <- paste0(
+#'   "https://www.arcgis.com/home/item.html",
+#'   "?id=10df2279f9684e4a9f6a7f08febac2a9"
+#' )
+#'
+#' arc_open(item_url)
```

Comment:
This resolves to a map server: 
https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer